src/libs/antares/jit/jit.hxx

```diff
@@ -68,11 +68,12 @@ void JIT::just_in_time_manager::unload_matrix_properly_from_memory(const Matrix<
 
     // - jit activated :
     //		If JIT (Just-In-Time) is activated, we have to unload data to keep the memory for
-    //the solver. 		The old width must only be kept when the matrix has fixed dimension.
+    // the solver. 		The old width must only be kept when the matrix has fixed dimension.
```

Comment:
clang was probably not passed on this file

Reply:
It was passed, but my clang-format is probably not exactly the same as the one run on branch develop.